ingest/bin/post_process_metadata.py

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env python3
+
+"""Reformat pandas DataTables for a pathogen build.
+
+Expecting one argument, a NCBI Virus metadata.tsv file
+"""
+# ===== Dependencies
+import argparse
+import os
+import sys
+
+import numpy as np
+import pandas as pd
+
+
+def parse_args():
+    # Main help command
+    parser = argparse.ArgumentParser(
+        description="Reformat a NCBI Virus metadata.tsv file for a pathogen build."
+    )
+    # Add first argument
+    parser.add_argument(
+        "--metadata", help="NCBI Virus metadata.tsv file.", required=True
+    )
+    parser.add_argument(
+        "--outfile",
+        help="Output file name, e.g. processed_metadata.tsv.",
+        required=True,
+    )
+
+    return parser.parse_args()
+
+
+# === Private methods
+
+
+def _set_strain_name(record):
+    """Check Isolate_s and Strain_s to find the strain name"""
```

Comment:
Either the docstring or code is wrong. `Isolate_s` is not checked.

Reply:
The context for this is `strain` is actually `Isolate_s` from genbank pull:

https://github.com/nextstrain/dengue/blob/7936e4c6ad4f4c62c1df822b7d7a01551b3c30dc/ingest/bin/genbank-url#L60-L61

But agree with you that the documentation is confusing, so will word it to "Check strain and strain_s to find the strain name"

Reply:
Resolved with https://github.com/nextstrain/dengue/pull/6/commits/5eb95767319a046b92bb5017f5042a06bc1d89d3